### ISSUE TITLE
New packet ID : 102

### ISSUE THREAD

**Issue by DeepSnowNeeL** [pulviscriptor/agario-client#43]
Hello guys, just to notice there is a new packet, id 102, it seems that you receive it when you die.
ex of data : 
[102, 8, 1, 18, 41, 8, 62, -14, 3, 36, 10, 13, 8, 10, 82, 9, 8, -70, 19, 16, 3, 24, 22, 32, 0, 18, 19, 8, 30, 16, -50, -33, 4, 24, -6, -101, 1, 32, -90, 85, 40, -69, 2, 48, -25, 5]

code corresponding in js :
 case 102:
                u = a.buffer.slice(d);
                p.core.proxy.forwardProtoMessage(u);
                break;

type in console : agarApp.core.proxy 
it looks like it's an API "Se.MiniclipAPI" to wss://web-live-v3-0.agario.miniclippt.com/ws

maybe some stats ?

**Comment by henopied** [comment]
Just wondering how do you track these things lol.

**Comment by pulviscriptor** [comment]
Its so hard to find free time to implement all those features.
I reserved https://github.com/pulviscriptor/agario-client/blob/master/account.js for wss://web-live-v3-0.agario.miniclippt.com/ws
I will try to implement it now, but if its too complicated, i will do it some time later.

**Comment by pulviscriptor** [comment]
I looked at code that works with wss://web-live-v3-0.agario.miniclippt.com/ws
As i see is that code have ping/pong/softPurchaseRequest/disconnect/loginRequest/updateUserSettings/activateBoostRequest/gameOver
So its just shop server (as i see). Not important stuff for agario-client (in my opinion). 
I made some code for it in `account.js` in case something would change and i will be implementing it.
Added 102 packet empty processor, no more warnings will be shown.
